deploy/kubernetes/console/templates/database.yaml

```diff
@@ -107,6 +165,9 @@ spec:
       imagePullSecrets:
       - name: {{.Values.dockerRegistrySecret}}
       {{- end }}
+      {{- if and (eq (printf "%s" .Values.kube.auth) "rbac") (.Capabilities.APIVersions.Has "rbac.authorization.k8s.io/v1") }}
+      serviceAccountName: "stratos-db"
```

Comment:
Should this be gated on `(.Values.console.pspEnabled) }}` as well, as the create `stratos-db` service is?

Reply:
Yes, it should